generators/app/templates/dotnet/Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN curl -SL https://raw.githubusercontent.com/Microsoft/MIEngine/getclrdbg-rele
     && ./GetClrDbg.sh $CLRDBG_VERSION \
     && rm GetClrDbg.sh<% } %>
 WORKDIR /app
+ENV ASPNETCORE_URLS http://*:<%= portNumber %>
```

Comment:
Could you wrap this in an if for dotnetVersion === 'RTM'? I believe this variable was not respected in RC2.